node_package/src/media/components/Page.jsx

```diff
@@ -70,15 +70,9 @@ export default connect(combineSelectors({
   textTracks: textTracks({
     file: prop('file'),
     defaultTextTrackFileId: prop('page.defaultTextTrackFileId')
-  }),
-  hasAutoplaySupport: has('autoplay support')
+  })
 }))(MediaPage);
 
-function willAutoplay(props) {
-  return props.page.autoplay !== false &&
-         props.hasAutoplaySupport;
-}
```

Comment:
The old logic enabled autoplay also when `page.autoplay` is undefined.

Reply:
Old logic restored